developer-docs/LANGUAGE_REFERENCE.md

```
@@ -104,15 +104,15 @@ lambi({"a": 1, "b": 2}) # 2
 lambi({1, 2, 3})        # 3
 ```
 
-### `range(n)`, `range(a, b)`, `range(a, b, s)`
+### `silsilo(n)`, `silsilo(a, b)`, `silsilo(a, b, s)`
 
-Creates a list of integers.
+Creates a lazy number range (`SdRange` — never materializes a list; O(1) `lambi()`, indexable, iterable).
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)